app/models/terminal_executor.rb

```diff
@@ -17,7 +17,7 @@ class TerminalExecutor
   SECRET_PREFIX = "secret://"
   HIDDEN_PREFIX = "hidden://"
   HIDDEN_TXT = "HIDDEN"
-  KILL_TIMEOUT = 1
+  KILL_TIMEOUT = (Rails.env.test? ? 1 : 5)
```

Comment:
hmm, still seems short. also not a fan of just random hardcoded config

Reply:
turned it into an env var 🤷‍♂